docs/docs/integrations/azure-devops.md

```diff
@@ -0,0 +1,13 @@
+# Azure Devops
+
+- Here is the [Azure DevOps Pipelines Task for Trivy][action]
+
+![trivy-azure](https://github.com/aquasecurity/trivy-azure-pipelines-task/blob/main/screenshot.png?raw=true)
+
+### [Microsoft Defender for container registries and Trivy][azure]
+
+The findings of the CI/CD scans are an enrichment to the existing registry scan findings by Qualys. Defender for Cloud's CI/CD scanning is powered by Aqua Trivy
```

Comment:
Could we add a bit more information -- not knowing what it is, it's a bit difficult to understand why I should look into it e.g. as an Azure user

Reply:
@AnaisUrlichs  I added more context from the azure blog

Reply:
Perfect 😄 thank you